### DISCUSSION TITLE
Don't recognize my Bluetooth device

### DISCUSSION THREAD

**Discussion by CanasCrespas** [dat-linux/community#51] [FAQ]
Hi, I am new on data linex an I'm trying to connect my headphones to the PC but it doesn't appear. I could connect it the first time, but now I can't.

Sorry I have another question, Can I put the folder ulin a dark mood. It doesn't allow me either 

Thanks.

**Comment by dat-linux** [comment]
Have you tried asking on Lubuntu support forum: https://discourse.lubuntu.me/c/support/6
